sdk/formrecognizer/ai-form-recognizer/src/transforms.ts

```diff
@@ -138,7 +138,7 @@ export function toFormTable(original: DataTableModel, readResults?: FormPage[]):
       boundingBox: toBoundingBox(cell.boundingBox),
       columnIndex: cell.columnIndex,
       columnSpan: cell.columnSpan || 1,
-      confidence: cell.confidence,
+      confidence: cell.confidence || 1,
```

Comment:
this sticks out to me. so if a confidence is unreported, we will fill it in as 100%?

Reply:
I remember that is suggested by the service team and also what we have at other places.

Reply:
.NET have default of 1.0 https://github.com/Azure/azure-sdk-for-net/blob/f3563578ec3d6ca654269f222a184974d8bf6d1f/sdk/formrecognizer/Azure.AI.FormRecognizer/src/DataTableCell_internal.cs#L21